api/DataTransfer.json

```diff
@@ -929,7 +929,11 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": "10",
+              "notes": [
+                "The property returns a <a href='https://developer.mozilla.org/docs/Web/API/DOMStringList'><code>DOMStringList</code></a>.",
```

Comment:
Isn't this the standard return value? Why would it need a compat note in that case then?

Reply:
no, the modern standard is to return some kind of an Array... well that is not so important.

Reply:
Looking into this a little bit more, it appears that the [HTML standard](https://html.spec.whatwg.org/multipage/dnd.html#the-datatransfer-interface) states that `types` should be a `FrozenArray<DOMString>`, which is basically the same as a `DOMStringList`, I'd say.  (Actually, it appears `DOMStringList` [was deprecated](https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#the-domstringlist-interface).)  As such, I feel that we can get rid of this note, unless there's a difference that affects developers?

Reply:
@vinyldarkscratch , it is the "same", if you are using `types.length` and `types[i]`, but in my case I was trying to use "types.indexOf('text/plain')" at some point (then I have decided, that it is better to write the code other way), so `Array.prototype.indexOf` was needed.

Reply:
Hmmm...yeah, if there's a big interface difference like the use of `indexOf`, it's probably noteworthy then.  I can't find this info in CanIUse, but hey, maybe we caught something they didn't!